src/components/video/data/service.js

```diff
@@ -12,30 +13,17 @@ const fetchAndAddTranscripts = async (transcriptUrls, player) => {
     .then(transcriptData => {
       const webVttData = convertToWebVtt(transcriptData);
       const webVttFileUrl = createWebVttFile(webVttData);
-
-      player.addRemoteTextTrack({
-        kind: 'subtitles',
-        src: webVttFileUrl,
-        srclang: lang,
-        label: lang,
-      }, false);
-
-      // We are only catering to English transcripts for now as we don't have the option to change
-      // the transcript language yet.
-      if (lang === 'en') {
-        player.vjstranscribe({
-          urls: [webVttFileUrl],
-        });
-      }
```

Comment:
[inform] abstracts the UI-layer out of service function. the service function should largely be about retrieving _data_, not necessarily incurring UI-related side effects like calling `player.addRemoteTextTrack`. These side effects ideally should be the responsibility of the component layer.